src/Tickets/Commerce/Gateways/PayPal/Webhooks/Events.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+namespace TEC\Tickets\Commerce\Gateways\PayPal\Webhooks;
+
+use TEC\Tickets\Commerce\Status as Commerce_Status;
+
+/**
+ * Class Events
+ *
+ * @since   TBD
+ *
+ * @package TEC\Tickets\Commerce\Gateways\PayPal\Webhooks
+ */
+class Events {
+	/**
+	 * Webhook Event name for a capture of completed payment.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	const PAYMENT_CAPTURE_COMPLETED = 'PAYMENT.CAPTURE.COMPLETED';
+
+	/**
+	 * Webhook Event name for a capture of denied payment.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	const PAYMENT_CAPTURE_DENIED = 'PAYMENT.CAPTURE.DENIED';
+
+	/**
+	 * Webhook Event name for a capture of refunded payment.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	const PAYMENT_CAPTURE_REFUNDED = 'PAYMENT.CAPTURE.REFUNDED';
+
+	/**
+	 * Webhook Event name for a capture of reversed payment.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	const PAYMENT_CAPTURE_REVERSED = 'PAYMENT.CAPTURE.REVERSED';
+
+	/**
+	 * Default mapping from PayPal Status to Tickets Commerce
+	 *
+	 * @since TBD
+	 *
+	 * @var array
+	 */
+	protected $default_map = [
+		self::PAYMENT_CAPTURE_COMPLETED => Commerce_Status\Completed::SLUG,
+		self::PAYMENT_CAPTURE_DENIED    => Commerce_Status\Denied::SLUG,
+		self::PAYMENT_CAPTURE_REFUNDED  => Commerce_Status\Refunded::SLUG,
+		self::PAYMENT_CAPTURE_REVERSED  => Commerce_Status\Reversed::SLUG,
+	];
+
+	/**
+	 * Gets the valid mapping of the webhook events.
+	 *
+	 * @since TBD
+	 *
+	 * @return array
+	 */
+	public function get_valid() {
+		return $this->default_map;
```

Comment:
Do we want to add a filter here? Thinking of third parties/extensions wanting to extend to a new event or something?

Reply:
Probably a good idea.